src/checks/key_without_value.rs

```diff
@@ -24,7 +24,10 @@ impl KeyWithoutValueChecker<'_> {
 impl Check for KeyWithoutValueChecker<'_> {
     fn run(&mut self, line: &LineEntry) -> Option<Warning> {
         if !line.raw_string.contains('=') {
-            Some(Warning::new(line.clone(), self.message(&line.raw_string)))
+            Some(Warning::new(
+                line.clone(),
+                self.message(&line.trimmed_string()),
```

Comment:
Why are these changes needed?

Reply:
Now we have the character `\ n` in the raw string.
I think this should not appear in the message.

Reply:
But if we will move back to `for (index, line) in reader.lines().enumerate()`, then it is not neccessary.

Reply:
Done